mobile/android/app/src/main/java/au/gov/vic/hobsonsbay/MainActivity.java

```diff
@@ -1,40 +1,35 @@
 package au.gov.vic.hobsonsbay;
 
-import android.os.Bundle;
-import org.devio.rn.splashscreen.SplashScreen;
 import com.facebook.react.ReactActivity;
 import com.facebook.react.ReactActivityDelegate;
-import com.facebook.react.ReactRootView;
-import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;
+import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint;
+import com.facebook.react.defaults.DefaultReactActivityDelegate;
 
 public class MainActivity extends ReactActivity {
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        SplashScreen.show(this, R.style.SplashScreenTheme);
-        super.onCreate(savedInstanceState);
-    }
+
   /**
    * Returns the name of the main component registered from JavaScript. This is used to schedule
    * rendering of the component.
    */
   @Override
   protected String getMainComponentName() {
-    return "HobsonsBay";
+    return "HoubsonBay";
```

Comment:
I've noticed the name change "HoubsonBay" but the actual name is "HobsonsBay" ... is this intentional?

Reply:
@jacinto-ciandt the name of the project I created was HoubsonBay. That's why I had to change the name.
Will Change it later on.